charts/datadog/templates/agent-clusterchecks-rbac.yaml

```
@@ -16,6 +16,7 @@ subjects:
 ---
 apiVersion: v1
 kind: ServiceAccount
+automountServiceAccountToken: {{ .Values.clusterChecksRunner.rbac.automountServiceAccountToken | default "true" }}
```

I'm wondering if `"true"` is correct for the default value. maybe `true` is enough

```suggestion
automountServiceAccountToken: {{ .Values.clusterChecksRunner.rbac.automountServiceAccountToken | default true }}
```

if if doesn't work you will need to do something like

```yaml
{{- if not .Values.clusterChecksRunner.rbac.automountServiceAccountToken }}
automountServiceAccountToken: false
{{- end }}
```

---

Thanks! Fixed